clang/test/Driver/lto.c

```diff
@@ -5,7 +5,7 @@
 // CHECK-COMPILE-ACTIONS: 2: compiler, {1}, ir
 // CHECK-COMPILE-ACTIONS: 3: backend, {2}, lto-bc
 
-// RUN: %clang -ccc-print-phases %s -flto 2> %t
+// RUN: %clang -ccc-print-phases %if target={{.*-windows-msvc.*}} %{ -fuse-ld=lld %} -flto %s 2> %t
```

Comment:
@MaskRay I went the `%if` route here.

Let me know if you prefer the sysroot route to handle systems that may not have `lld` installed as was done [here](https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/lto.c#L51) and [here](https://github.com/llvm/llvm-project/tree/main/clang/test/Driver/Inputs/basic_cross_linux_tree/usr/x86_64-unknown-linux-gnu/bin). 

As noted above the `*-windows-msvc` target currently does not error if `lld-link.exe` does not exist but let me know if you would rather I go the `sysroot` route for the `-ccc-print-phases` tests here.